Modules/Sources/FormattableContentKit/NotificationContentFactory.swift

```diff
@@ -6,6 +6,7 @@ private enum Constants {
     static let Ranges = "ranges"
 }
 
+// FIXME: This is a WordPressData notification!
```

Comment:
This should be fixable by moving some bits back to the app target. I might have been too quick in moving types...

Reply:
Actually... I'm not sure it's a WordPressData Notification... Anyway, I got rid of `Notification` here altogether in https://github.com/wordpress-mobile/WordPress-iOS/pull/24354